### ISSUE TITLE
Installing KlipperScreen via KIAUH causes /dev/null permissions errors

### ISSUE THREAD

**Issue by rdbahm** [dw-0/kiauh#448]
### Linux Distribution

- MainsailOS 1.3.2 arm64 on Raspberry Pi 4 (tried on multiple RPi4s)
- Raspberry Pi OS Bullseye arm64 on Raspberry Pi 4
- Debian Bullseye amd64 on virtual machine (for verification that this isn't a weird RPi issue)

### What happened

I chose the "Install" "KlipperScreen" option in KIAUH, and selected "yes" for "Install as a service". Once complete, errors began cropping up in the general format:
- fatal: open /dev/null or dup failed: Permission denied
- /home/pi/kiauh/scripts/crowsnest.sh: line 142: /dev/null: Permission denied
- -bash: /dev/null: Permission denied

A screenshot of the terminal after this issue (screenshot is from the Debian Bullseye VM)
![image](https://github.com/dw-0/kiauh/assets/4553626/42cf61a0-f44d-4d95-9800-4bb6c90e5789)****

### What did you expect to happen

When installing KlipperScreen with the [KlipperScreen manual install documentation](https://klipperscreen.github.io/KlipperScreen/Installation/#klipperscreen-installation) (which, as far as I can tell, uses the very same install script cloned from the same repo), the issue does not occur.

In short, I would expect to retain access to /dev/null.

Same VM as previous screenshot, but instead of installing through KIAUH, installed with the KlipperScreen install script pulled directly from its repository. (This was done with the help of snapshots, to keep the underlying environment identical):
![image](https://github.com/dw-0/kiauh/assets/4553626/d5b5222d-1f59-4119-b727-65d3ea0cd768)

### How to reproduce

1. On Debian Bullseye, install KIAUH, following its installation instructions.
2. Once in KIAUH, install KlipperScreen.
3. When prompted, select "Yes" for "Install as a service."
4. Return to the KIAUH main menu to see the first errors.

**Comment by rdbahm** [comment]
While investigating the problem I was having to write up the Issue, I found a [seemingly related issue](https://github.com/KlipperScreen/KlipperScreen/issues/1297) in KlipperScreen, which occurred when KlipperScreen's uninstall script was run, even when KIAUH was not involved.

Digging down that rabbithole, while I don't understand why the issue doesn't seem to occur when installing KlipperScreen directly, but does through KIAUH, I do think I have a workaround:

1. cd to your KlipperScreen repository.
2. Run scripts/Uninstall.sh
3. Reboot
4. cd to your KlipperScreen repository.
5. Run git checkout v0.3.9
6. Run scripts/KlipperScreen-install.sh

At present, it looks like this issue may have been due to a regression in KlipperScreen (see the other issue for details), but I've left it open due to the difference in behavior between KIAUH-installed KlipperScreen and manually-installed KlipperScreen. Hoping a second set of eyes can help make sense of this.

**Comment by rdbahm** [comment]
Seems to be fixed with KlipperScreen commit 9052481af3195e3ebdc217f3d13bbe915a3a83c4.

Sorry for the false alarm over here. :)
